lib/ClangImporter/ImportDecl.cpp

```diff
@@ -334,6 +332,20 @@ getSwiftStdlibType(const clang::TypedefNameDecl *D,
     *IsError = true;
     return std::make_pair(Type(), "");
   }
+
+  if (CTypeKind == MappedCTypeKind::VaList) {
+    const llvm::Triple &T = ClangCtx.getTargetInfo().getTriple();
+    // Darwin AArch64 uses void * for the va_list type, while AAPCS64 uses a
+    // custom type.  Once the TODO below is addressed, this check should not be
+    // needed.
+    bool IsAAPCS64 = T.getArch() == llvm::Triple::AArch64 && !T.isOSDarwin();
+    // TODO compare this against the size of the SwiftType rather than the
+    // void * type.
+    if (!IsAAPCS64 &&
+        (ClangTypeSize != ClangCtx.getTypeSize(ClangCtx.VoidPtrTy)))
```

Comment:
@jrose-apple I couldn't see the equivalent of `getTypeSize` in the swift `ASTContext`, perhaps I missed something?  Is there an easy way to address that TODO?

Reply:
Not really. Swift doesn't bother measuring sizes until IR generation time, and even then you won't be able to do it under certain conditions. I think it's okay to just explicitly leave AAPCS64 out of the impedance check.

Reply:
Okay, Ill probably move this back into the switch then